.github/workflows/generate-sdk.yml

```diff
@@ -88,6 +89,7 @@ jobs:
         uses: actions/download-artifact@v2
       - name: Add and Commit files
         run: |
+          echo ${{ github.head_ref }}
```

Comment:
This can be removed, it was only being used to debug a pipeline issue.

Reply:
remved